lib/lotus/generators/application/app.rb

```diff
@@ -47,6 +47,7 @@ def start
             'config/routes.rb.tt'               => 'config/routes.rb',
             'views/application_layout.rb.tt'    => 'app/views/application_layout.rb',
             'templates/application.html.erb.tt' => 'app/templates/application.html.erb',
+            'lib/tasks/env.rake.tt'             => 'lib/tasks/env.rake'
```

Comment:
Are you against expanding this abbr? `env => environment`, I think it'll avoid future confusions if the file has the same name of the task :)